lib/cisco_node_utils/bgp_af.rb

```diff
@@ -398,9 +422,15 @@ def dampening=(damp_array)
         CiscoLogger.debug("Dampening 'dampening #{damp_array.join(' ')}''")
       elsif route_map.is_a? String
```

Comment:
Geert, would you mind changing this line to 'else' and removing the unreachable 'else fail ArgumentError' below?